source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,263 @@
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': {
+      production: 'bower_components/handlebars/handlebars.js'
+    }
+  }
+});
+```
+
+Alternatively, if you want to exclude the built-in asset from being
+automatically included in `vendor.js`, you can set its value to `false`:
+
+```Brocfile.js
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': false
+  }
+});
+
+```
+
+_Note: The built-in assets are required dependencies needed by the environment
+to run your app. If you use the above method to specifically exclude
+some, you should still be including them in some other way._
+
+##### Test Assets
+
+You may have additional libraries that should only be included when running tests (such as qunit-bdd or sinon). These can be imported into your app in your Brocfile.js:
+
+```Brocfile.js
+var EmberApp = require('ember-cli/lib/broccoli/ember-app'),
+    isProduction = EmberApp.env() === 'production';
+
+var app = new EmberApp();
+
+if ( !isProduction ) {
+    app.import( app.bowerDirectory + '/sinonjs/sinon.js', { type: 'test' } );
+    app.import( app.bowerDirectory + '/sinon-qunit/lib/sinon-qunit.js', { type: 'test' } );
+}
+
+module.exports = app.toTree();
+```
+
+**Notes:**
+- Be sure to pass `{ type: 'test' }` as the second argument to `app.import`.  This will ensure that your libraries are compiled into the `test-support.js` file.
+
+#### Styles
+
+##### Static CSS
+
+Provide the asset path as the first argument:
+
+```Brocfile.js
+app.import('bower_components/foundation/css/foundation.css');
+```
+
+All style assets added this way will be concatenated and output as `/assets/vendor.css`.
+
+##### Dynamic Styles (SCSS, LESS, etc)
+
+The vendor trees that are provided upon instantiation are available to your dynamic style files.  Take the following example (in `app/styles/app.scss`):
+
+```scss
+@import "bower_components/foundation/scss/normalize.scss";
+```
+
+#### Other Assets
+
+##### Using app.import()
+
+All other assets like images or fonts can also be added via `import()`. By default, they
+will be copied to `dist/` as they are.
+
+```Brocfile.js
+app.import('bower_components/font-awesome/fonts/fontawesome-webfont.ttf');
+```
+
+This example would create the font file in `dist/font-awesome/fonts/fontawesome-webfont.ttf`.
+
+You can also optionally tell `import()` to place the file at a different path.
+The following example will copy the file to `dist/assets/fontawesome-webfont.ttf`.
+
+```Brocfile.js
+app.import('bower_components/font-awesome/fonts/fontawesome-webfont.ttf', {
+  destDir: 'assets'
+});
+```
+
+If you need to load certain dependencies before others, you can set the `prepend` property equal to `true` on the second argument of `import()`. This will prepend the dependency to the vendor file instead of appending it, which is the default behavior.
+
+```Brocfile.js
+app.import('bower_components/es5-shim/es5-shim.js', {
+  type: 'vendor',
+  prepend: true
+});
+```
+
+##### Using broccoli-funnel
```

Comment:
If this isn't installed by Ember CLI by default, I'm not sure it's something we should cover in the Guides.